### PR TITLE
Added GlobalActions Controller and Handler. Adjusted naming to match …

### PR DESCRIPTION
**Pull request by pdimitratos**
…conventions.

### CODE REVIEW

File: src/Sia.Gateway/Sia.Gateway.csproj [hunk 1]
```diff
@@ -37,7 +37,6 @@
     <ProjectReference Include="..\Sia.Data.Incident\Sia.Data.Incidents.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <Folder Include="Controllers\Playbook\" />
```
Comment (pdimitratos) [inline src/Sia.Gateway/Sia.Gateway.csproj]: I have no idea why this line went away, but it doesn't seem to have hurt anything.

File: src/Sia.Gateway/Requests/Playbook/GlobalActions/GetGlobalActions.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+using Sia.Gateway.Requests.Playbook;
+using Sia.Shared.Authentication;
+using Sia.Shared.Authentication.Http;
+using Sia.Shared.Requests;
+
+namespace Sia.Gateway.Requests
+{
+    public class GetGlobalActionsRequest : AuthenticatedRequest<IEnumerable<Action>>
+    {
+        public GetGlobalActionsRequest(AuthenticatedUserContext userContext)
```
Comment (macrobug) [inline src/Sia.Gateway/Requests/Playbook/GlobalActions/GetGlobalActions.cs]: ctor seems redundant

File: src/Sia.Gateway/Requests/Playbook/GlobalActions/GetGlobalActions.cs [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+            : base(userContext)
+        {
+        }
+    }
+
+    public class GetGlobalActionsHandler : PlaybookProxyHandler<GetGlobalActionsRequest, IEnumerable<Action>>
+    {
+        public GetGlobalActionsHandler(HttpClientLookup clientFactory) : base(clientFactory)
```
Comment (macrobug) [inline src/Sia.Gateway/Requests/Playbook/GlobalActions/GetGlobalActions.cs]: ctor seems redundant

File: src/Sia.Gateway/Requests/Playbook/GlobalActions/GetGlobalActions.cs (hunk 2 above)
Comment (pdimitratos) [reply src/Sia.Gateway/Requests/Playbook/GlobalActions/GetGlobalActions.cs]: It doesn't compile without this constructor.

**Pull request merged**